test/integration/targets/find/tasks/main.yml

```diff
@@ -375,5 +375,61 @@
       - '"{{ remote_tmp_dir_test }}/astest/.hidden.txt" in astest_list'
       - '"checksum" in result.files[0]'
 
+# Test permission error is correctly handled by find module
+- vars:
+    test_dir: /tmp
```

Comment:
Use a subdirectory under `/tmp` and then clean that directory up at the end of the block.